controller:
  ros__parameters:
    log_level: "debug"
    frame_center:
      x: 960
      y: 580 # A bit lower in order to see the plane better and have more altitude

    PID_Aileron:
      P: 0.0
      I: 0.0
      D: 0.0
      FF: 0.0
    PID_Elevator:
      P: 0.0
      I: 0.0
      D: 0.0
      FF: 0.0
    PID_Rudder:
      P: 0.0
      I: 0.0
      D: 0.0
      FF: 0.0
    PID_Thrust:
      P: 0.0
      I: 0.0
      D: 0.0
      FF: 0.0
